includes/utilities.inc

```diff
@@ -364,7 +364,7 @@ function islandora_solr_prepare_solr_results($solr_results) {
       }
 
       // Implode.
-      $value = is_array($value) ? implode(", ", $value) : $value;
+      $value = is_array($value) ? implode(filter_xss($separator, array('br')), $value) : $value;
```

Comment:
If the separator is configurable does the `filter_xss` tags allowed need to be more broad than just `br`?